tests/test_twitter_ads_automatic_fields.py

```diff
@@ -28,6 +28,18 @@ def test_run(self):
                                             'cards_video_direct_message', 'accounts_daily_report', 'campaigns_daily_report', 
                                             'targeting_tv_markets', 'targeting_tv_shows'}
 
+        # Set page_size to 1000 for following streams because these streams contain more than 40000 records.
+        # So, page_size of 200 get a lot of time to get all records.
+        self.run_test(streams_to_test={"targeting_locations", "targeting_conversations"}, page_size=1000)

```

Comment:
To test config param, how about taking 990?

Reply:
We got `ConnectionResetError` for these 2 streams for `page_size = 200`. That's why we increased page_size to the maximum value to reduce the possibility of `ConnectionResetError`.